apis/core/lib/domain/job/create.ts

```diff
@@ -74,9 +79,75 @@ export async function createPublishJob({
     publishedTo: metadata?.pointer.out(schema.workExample).term,
   })
 
+  const cubeId = isCsvProject(project)
+    ? organization.createIdentifier({
+      cubeIdentifier: project.cubeIdentifier,
+    })
+    : project.sourceCube
+
+  const dataset = await store.getResource<Dataset>(project.dataset.id)
+  const workExamples = prepareWorkExamples(cubeId, job, dataset)
+  workExamples.forEach(workExample => job.addWorkExample(workExample))
+
   return jobPointer
 }
 
+export function prepareWorkExamples(cubeId: NamedNode, job: Job.PublishJob, dataset: Dataset): Job.WorkExampleInput[] {
+  const workExamples = []
+
+  if (env.has('TRIFID_UI')) {
+    workExamples.push({
+      url: $rdf.namedNode(lindasWebQueryLink(cubeId, job.revision, job.publishGraph)),
+      encodingFormat: $rdf.literal('Application/Sparql-query'),
```

Comment:
@l00mi I copied this "format" from opendata.swiss but I remember you telling me that it was BS. What should I use for the link to LINDAS query endpoint instead?

Reply:
`application/sparql-query` is the request type for SPARQL queries.

should this be the response format instead? a query endpoint can do content negotiation so maybe it multiple should be listed? `application/sparql-results+json` and `application/sparql-results+xml`?

Reply:
I'm open to anything. The only thing to know is that I use this value in the CKAN endpoint to determine the "format" property (SERVICE, HTML, etc.). So I need different values for visualize and the query endpoint.